genesyscloud/webdeployments_configuration/resource_genesyscloud_webdeployments_configuration_test.go

```diff
@@ -1157,3 +1164,27 @@ func verifyConfigurationDestroyed(state *terraform.State) error {
 
 	return nil
 }
+
+func cleanupWebDeploymentsConfiguration() error {
+	config, err := provider.AuthorizeSdk()
+	if err != nil {
+		return err
+	}
+	deploymentsAPI := platformclientv2.NewWebDeploymentsApiWithConfig(config)
+
+	configurations, resp, getErr := deploymentsAPI.GetWebdeploymentsConfigurations(false)
+	if getErr != nil {
+		return util.BuildWithRetriesApiDiagnosticError(resourceName, fmt.Sprintf("failed to get page of configurations: %v", getErr), resp)
+	}
+
+	for _, configuration := range *configurations.Entities {
+		if configuration.Name != nil && strings.HasPrefix(*configuration.Name, "Test Configuration ") {
```

Comment:
I would pass the prefix value in as a parameter, so then it doesn't become the test's job to name the configuration correctly for this cleanup function to work